lib/engine/game/g_1860.rb

```diff
@@ -388,7 +388,7 @@ def check_nationalize?
       end
 
       def get_or_revenue(info)
-        info.dividend.kind == 'withhold' ? 0 : info.revenue
+        !info.dividend.is_a?(Action) || info.dividend.kind == 'withhold' ? 0 : info.revenue
```

Comment:
Isn't info.dividend always going to be an action (either the official dividend action, or the previous action)? I'm still not sure how this works.